packages/drafts-realm/CyclingMileageLog/9fc98a27-ad5c-450d-843f-9c55730f0122.json

```diff
@@ -0,0 +1,31 @@
+{
+  "data": {
+    "type": "card",
+    "attributes": {
+      "target": 1500,
+      "entries": [
+        {
+          "date": "2024-05-03",
+          "origin": "Bandung Indah Plaza",
+          "destination": "Kamojang Mountain",
+          "distance": 65
```

Comment:
Since people use miles too it would be good to add the unit setting (km or mile)